test/README.md

```diff
@@ -0,0 +1,53 @@
+# Tests
+
+## Structures
+
+There are 2 suites of tests here: [`e2e`](./e2e) and [`osd`](./osd). 
+
+* [`e2e`](./e2e)
+  
+  The tests in this directory are executed using the operator-sdk's test command against an OCP environment, and will be exectued everytime when a PR is opened.
```

Comment:
The operator-sdk e2e tests can be run against any openshift cluster, including osd